modcluster/runtime/src/main/java/org/wildfly/swarm/mod_cluster/runtime/ModclusterParserFactory.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2015-2016 Red Hat, Inc, and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.swarm.mod_cluster.runtime;
+
+import org.jboss.dmr.ModelNode;
+import org.jboss.staxmapper.XMLElementReader;
+import org.wildfly.extension.mod_cluster.ModClusterExtension;
+import org.wildfly.swarm.spi.runtime.AbstractParserFactory;
+
+import javax.xml.namespace.QName;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Stuart Douglas
+ */
+public class ModclusterParserFactory extends AbstractParserFactory {
```

Comment:
Does that mean I can remove the whole runtime module?

I have had this sitting 90% complete for the last few weeks, I did not check if anything had changed in the meantime.